asapdiscovery-docking/asapdiscovery/docking/workflows/large_scale_docking.py

```diff
@@ -363,21 +460,59 @@ def large_scale_docking_workflow(inputs: LargeScaleDockingInputs):
     else:
         bleach_columns = False
 
+    # keep everything not just hits
     result_df = rename_output_columns_for_manifold(
         scores_df,
         inputs.target,
         [DockingResultCols],
         bleach_columns=bleach_columns,
         manifold_validate=True,
         drop_non_output=True,
-        allow=[DockingResultCols.LIGAND_ID.value],
+        allow=[
+            DockingResultCols.LIGAND_ID.value,
+            DockingResultCols.HTML_PATH_POSE.value,
+            DockingResultCols.HTML_PATH_FITNESS.value,
+        ],
     )
 
     result_df.to_csv(output_dir / "docking_results_final.csv", index=False)
 
     if inputs.postera_upload:
         logger.info("Uploading results to Postera")
+
         postera_uploader = PosteraUploader(
             settings=PosteraSettings(), molecule_set_name=inputs.postera_molset_name
         )
-        postera_uploader.push(result_df)
+        # push the results to PostEra, making a new molecule set if necessary
+        result_df, molset_name, made_new_molset = postera_uploader.push(result_df)
+
+        if made_new_molset:
+            logger.info(f"Made new molecule set with name: {molset_name}")
+        else:
+            molset_name = inputs.postera_molset_name
+
+        logger.info("Uploading artifacts to PostEra")
+
+        # make an uploader for the poses and upload them
+
+        artifact_columns = [
```

Comment:
Visualisations are being made and pushed for all compounds as take < 1 sec per compound.

Reply:
great!